climada/hazard/test/test_base.py

```diff
@@ -41,7 +41,7 @@
 """
 Directory for writing (and subsequent reading) of temporary files created during tests.
 """
-HAZ_TEST_MAT :Path = Path(hazard_test.__file__).parent.joinpath('data', 'atl_prob_no_name.mat')
+HAZ_TEST_MAT :Path = get_test_file('atl_prob_no_name', file_format='matlab')
```

Comment:
Any reason to sometimes use the `HAZ_TEST_MAT :Path =` style and sometimes `HAZ_TEST_MAT =` ?

Reply:
No reason, except that no type annotation has been changed in this PR.